app/labor/badge_rewarder.rb

```diff
@@ -62,14 +62,27 @@ def self.award_tag_badges
 
   def self.award_contributor_badges_from_github(since = 1.day.ago, message_markdown = "Thank you so much for your contributions!")
     client = Octokit::Client.new
-    badge = Badge.find_by(slug: "dev-contributor")
-    ["thepracticaldev/dev.to", "thepracticaldev/DEV-ios", "thepracticaldev/DEV-Android"].each do |repo|
+    repos = ["thepracticaldev/dev.to", "thepracticaldev/DEV-ios", "thepracticaldev/DEV-Android"]
+    repos.each do |repo|
       commits = client.commits repo, since: since.iso8601
       authors_uids = commits.map { |commit| commit.author.id }
       Identity.where(provider: "github", uid: authors_uids).find_each do |i|
-        BadgeAchievement.where(user_id: i.user_id, badge_id: badge.id).first_or_create(
-          rewarding_context_message_markdown: message_markdown,
-        )
+        user_commits = repos.map { |rep| client.commits(rep, author: i.auth_data_dump["info"]["nickname"]) } # commits from all DEV repos.
+        user_commits_count = user_commits.map(&:count).sum # sum of commits from all DEV repos.
+        milestones = [
+          { count: 1, slug: "dev-contributor" },
+          { count: 32, slug: "dev-contributor-bronze" },
```

Comment:
Based on the discussion [here](https://github.com/thepracticaldev/dev.to/issues/4002#issuecomment-530756146). I think it would be a good idea to lower bar to 1,16,32..
What do you think @benhalpern ?

Reply:
Your thoughts @benhalpern ?

Reply:
Hey @FeruzOripov sorry about the delay in response.

Yes I think lowering to 16 makes sense. It's a minor thing, I think this PR is pretty much ready to go. We have the designs close to finalized as well.